src/htdocs/monitoring/operations/_station.php

```diff
@@ -114,6 +120,10 @@
             ?></td>
           </tr>
           <tr>
+            <th scope="row">Network</th>
+            <td><?php echo FDSNlink($properties['network_code']);?></td>
```

Comment:
Because of how the template executes, the function would be defined twice and generate an error.

I recommend inlining the formatting instead of using a function:

```
<td><?php
  $net = $properties['network_code'];
  echo '<a href="http://www.fdsn.org/networks/detail/' . $net . '"' .
       target="_blank" title="FDSN Network ' . $net . '">' . $net . '</a>';
?></td>
```